addons/eu/en16931/bill.go

```diff
@@ -113,6 +122,21 @@ func validateBillInvoiceTax(value any) error {
 	)
 }
 
+func validateParty(value any) error {
```

Comment:
Does this need to be namespace for the context?

```suggestion
func validateBillInvoiceParty(value any) error {
```